### DISCUSSION TITLE
Repository comments

### DISCUSSION THREAD

**Discussion by romw314** [community/community#62099] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

I have an suggestion for GitHub.
I want to post a comment for a repo and I don't know how.
Does GitHub support it?
Or should I open an issue on the repo.

**Comment by Hsin-Hung** [comment, marked as answer]
As of now, I think the only way you can comment on a repo is by opening an Issue or starting a Discussion if that is enabled.

**Comment by romw314** [comment]
😒

**Comment by romw314** [comment]
So I should open an issue when I want to post something about it.
But this can be against any code of conduct, etc.

**Reply by Hsin-Hung** [reply]
You should first look at the repo's contributing guide (CONTRIBUTING.md), which contains the issue/pr rules and code of conduct they want you to follow before contributing. They might also mention additional communication channels for general comments or discussions.

If they didn't provide any contributing guides, you can just post your comment through an issue. You can't violate any code of conduct if they don't have one in the first place. Hope this helps!
